armi/reactor/reactors.py

```diff
@@ -1811,547 +1815,6 @@ def createAssemblyOfType(self, assemType=None, enrichList=None, cs=None):
-            yBlockLoc += blockHeight
-            yBlockHeights.append(yBlockLoc)
-
-            # Add location, block heights, and axial mesh points to ordered set
-            yBlockAxMesh.add((yBlockCenterLoc, blockHeight, b.p.axMesh))
-
-        # Add the block heights, block number of axial mesh points on the far right of the plot.
-        if isLastAssem and showBlockAxMesh:
-            xEndLoc = 0.5 + xAssemEndLoc
-            for bCenter, bHeight, axMeshPoints in yBlockAxMesh:
-                axis.text(
-                    xEndLoc,
-                    bCenter,
-                    "{} cm ({})".format(bHeight, axMeshPoints),
-                    fontsize=10,
-                    ha="left",
-                )
-
-        return xBlockLoc, yBlockHeights, yBlockAxMesh
-
-    def getAssemTypeLetters(self):
-        """
-        Builds a list of unique capital letters for each assembly.
-
-        Tries with the first letter of each word, then goes to just continuous letters.
-        If nothing unique, just pulls from the alphabet.
-
-        Returns
-        -------
-        assemTypeLetters : dict
-            keys are assembly types, vals are capital letters.
-        """
-
-        assemTypeLetters = {}
-        doneLetters = []
-        candidate = None
-        alphabet = frozenset([chr(units.ASCII_LETTER_A + i) for i in range(26)])
-        for a in self:
-            aType = a.getType()
-
-            if aType not in assemTypeLetters:
-                # build list of candidate letters. Start with first letter,
-                # then first letter of second word, then all following letters.
-                candidates = [aType[0]]
-                for word in aType.split()[1:]:
-                    # add first letter of each word
-                    candidates.append(word[0])
-                candidates.extend(aType.replace(" ", "")[1:])  # no spaces allowed
-                # add in remaining letters in alphabet
-                done2 = frozenset(
-                    doneLetters
-                )  # frozensets allow - to be a difference operator. Very nice.
-                candidates.extend([a for a in alphabet - done2])
-
-                # find a unique candidate letter
-                for candidate in candidates:
-                    candidate = candidate.upper()
-                    if candidate not in doneLetters:
-                        doneLetters.append(candidate)
-                        break
-
-                if not candidate:
-                    runLog.error(
-                        "Cannot determine type letter of {0} assemblies. Are there more "
-                        "than 26 assembly types? If so, upgrade assembly type ID system."
-                        "Done letters are: {1}".format(aType, doneLetters)
-                    )
-                    raise RuntimeError("Failed to determine assembly type letter.")
-                assemTypeLetters[aType] = candidate
-
-        return assemTypeLetters
-
```

Comment:
Mmmmmmmmm

Reply:
yeah, everything was basically just moved (with some refactor on plot face map). the only thing that I actually got to delete was getAssemTypeLetters